### ISSUE TITLE
NodeFactory missing SetHidden() method

### ISSUE THREAD

**Issue by mbreissi** [IOT-DSA/sdk-dslink-dotnet#43]
Need the ability to create hidden nodes as provided by other SDKs

**Comment by mbreissi** [comment]
Suggested implementation: 
```
	    public NodeFactory SetHidden(bool hidden)
	    {
		    SetConfig(ConfigType.Hidden, new Value(hidden));
		    return this;
	    }
```

**Comment by logangorence** [comment]
This is fixed via #49.
